### ISSUE TITLE
ESR115 links on /firefox/all/ are 404 for /sat/ and /skr/ languages

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15437]
## Description

In https://github.com/mozilla/bedrock/pull/15390 we added Firefox ESR 115 download links to the `/firefox/all/` page. Our download test job has identified a couple of languages where these links are 404:

- https://www.mozilla.org/en-US/firefox/all/desktop-esr/win64/skr/
- https://www.mozilla.org/en-US/firefox/all/desktop-esr/win64/sat/

Test failure: https://github.com/mozilla/bedrock/actions/runs/11665748254/job/32479161086

**Comment by alexgibson** [comment]
Full list of 404 URLs:

```
https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=win64&lang=sat
https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=win64&lang=skr
https://download.mozilla.org/?product=firefox-esr115-msi-latest-ssl&os=win64&lang=sat
https://download.mozilla.org/?product=firefox-esr115-msi-latest-ssl&os=win64&lang=skr
https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=win64-aarch64&lang=skr
https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=win64-aarch64&lang=sat
https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=win&lang=sat
https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=win&lang=skr
https://download.mozilla.org/?product=firefox-esr115-msi-latest-ssl&os=win&lang=sat
https://download.mozilla.org/?product=firefox-esr115-msi-latest-ssl&os=win&lang=skr
https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=osx&lang=sat
https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=osx&lang=skr
https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=linux64&lang=sat
https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=linux64&lang=skr
https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=linux&lang=sat
https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=linux&lang=skr
```

**Comment by alexgibson** [comment]
I inquired about this in Slack, and it sounds like both `skr` and `sat` were added post ESR 115, so it's expected that the links don't work.

- (skr) https://bugzilla.mozilla.org/show_bug.cgi?id=1900150
- (sat) https://bugzilla.mozilla.org/show_bug.cgi?id=1853054

So, I guess we should omit the button for these languages on the bedrock side?

I also wonder (thinking about new languages that might be added in the future), if it's possible to fall back to en-US somehow if an ESR115 build does not exist?
